src/opentimelineio/track.h

```diff
@@ -17,6 +17,7 @@ class Track : public Composition
     {
         static auto constexpr video = "Video";
         static auto constexpr audio = "Audio";
+        static auto constexpr data = "Data";
```

Comment:
I'm not too familiar with data tracks in NLEs. Do you know of a good explanation of them I could read? Is that same term used in other NLEs besides Avid Media Composer?